src/snowflake/snowpark/modin/plugin/PANDAS_CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ### Bug Fixes
 - Fixed bug when creating a DataFrame with a dict of Series objects.
+- Fixed incorrect return type in `qcut` when given `Series` input and improved error checking logic.
```

Comment:
It seems that this line is unrelated to this PR?

Reply:
+1

Reply:
Yeah, I moved some lines in the changelog to put them in the correct sections, this one was in the 1.15.0a1 release section but it was not a part of that release.